cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
         )
```

Comment:
This validation method might be better placed in the model. It's generally a good practice to keep business logic in the model. This allows you to reuse the same logic in different parts of your application, and it keeps the serializer focused on serialization tasks.

Reply:
I have validation in the model, what means validate on DB side. And I have validation in serializer, what means validate before DB side.